model/tensorflow/setup.py

```diff
@@ -18,7 +18,7 @@
 with open(os.path.join(self_path, "README.md"), "r", encoding="utf-8") as f:
     readme = f.read()
 
-INSTALL_REQUIRES = ["tensorflow>=2.0.0"] + (
+INSTALL_REQUIRES = ["tensorflow>=2.0.0", "scikit-learn>=0.21.2"] + (
```

Comment:
It looks like scikit is only being used for testing, can we move it to the `tests_require` section

```python
        "Programming Language :: Python :: Implementation :: PyPy",
    ],
    install_requires=INSTALL_REQUIRES,
    tests_require=["scikit-learn>=0.21.2"],
    packages=find_packages(),
    entry_points={
```